src/applications/static-pages/school-resources/constants/announcements.json

```diff
@@ -1,4 +1,11 @@
 [
+    {     
+    "name": "Changes to the “48 Month Rule” for VR&E and EDU Beneficiaries",
```

Comment:
Since this is JSON data, I think you will need to escape the double quotes within the name with a backslash \